pkg/frontend/transport/handler.go

```diff
@@ -409,6 +409,7 @@ func (f *Handler) reportQueryStats(r *http.Request, source, userID string, query
 		"split_queries", splitQueries,
 		"status_code", statusCode,
 		"response_size", contentLength,
+		"samples_processed", numScannedSamples,
```

Comment:
Why we don't call it `samples_scanned`?  queryScannedSamples metric is named `cortex_query_samples_scanned_total` so we should be consistent here